package.json

```diff
@@ -1,20 +1,33 @@
 {
   "name": "react-firebase",
-  "version": "0.1.0",
+  "version": "2.0.0",
   "description": "React bindings for Firebase",
-  "main": "./lib/index.js",
+  "author": "Simen Brekken <simen@unfold.no> (http://github.com/unfold)",
```

Comment:
Our current convention is to use company as author
- [webpack-serve/package.json#L27](https://github.com/unfold/webpack-serve/blob/master/package.json#L27)
- [fetch-data/package.json#L27](https://github.com/unfold/fetch-data/blob/master/package.json#L27)
- [config-variables/package.json#L21](https://github.com/unfold/config-variables/blob/master/package.json#L21)